File: docs/app/samplecode/3/4.js
const team1 = _.slice(response['participantIdentities'], 0, 5)
const team1Names = _.map(team1, (p) => { return p['player']['summonerName'] })

const team2 = _.slice(response['participantIdentities'], 5, 10)
const team2Names = _.map(team2, (p) => { return p['player']['summonerName'] })

console.log(`*** Match id ${matchId} featured`)
console.log(`team 1: ${ _.join(team1Names, ', ')} `)
console.log("versus")
console.log(`team 2: ${ _.join(team2Names, ', ')} `)

File: docs/app/samplecode/3/4.rb
team1 = response['participantIdentities'].slice(0..4)
team1_names = team1.map { |p| p['player']['summonerName'] }

team2 = response['participantIdentities'].slice(5..9)
team2_names = team2.map { |p| p['player']['summonerName'] }

puts "*** Match id #{match_id} featured"
puts "team 1: #{team1_names.join(', ')}"
puts "versus"
puts "team 2: #{team2_names.join(', ')}"
